content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx

```diff
@@ -229,7 +266,7 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
                 </AlignCenter>
               </div>
               <span>
-                <NextLink href={`${guideTypeUrl}#${stopNumber}`} shallow={true}>
```

Comment:
`shallow` doesn't have any effect when changing NextJS pages (which this is) so removed it